### ISSUE TITLE
nREPL weirdness when evaluating ns form multiple times

### ISSUE THREAD

**Issue by borkdude** [babashka/nbb#80]
``` clojure
(ns foo
  (:require ["dayjs$default" :as dayjs]
            ["moment$default" :as moment]))

dayjs
moment

(prn (dayjs))
(prn (moment))
```

Try to evaluate this from nREPL. 

Reproduce the error without nREPL:

```
(ns foo
  (:require ["dayjs$default" :as dayjs]
            ["moment$default" :as moment]))


(ns foo
  (:require ["dayjs$default" :as dayjs]
            ["moment$default" :as moment]))

(prn (dayjs))
(prn (moment))
```

Evaluating the same ns form twice results in these errors.

**Comment by borkdude** [comment]
The error seems to happen when you eval the ns form more than once.

**Comment by borkdude** [comment]
Should be fixed in v0.0.98 which is currently being built + deployed on CI.
